### PR TITLE
Added changelog for v1.0.0. Updated package version to 1.0.0

### LINKED ISSUE #402: Changelog for 1.0.0

### PR DESCRIPTION
**Pull request by SwinX**
resolves #402

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,33 @@
...
+* Удалены старые моки: `test-node`, `test-logger`, `test-file-system`. `mock-node` и `mock-logger` могут быть найдены в пакете `mock-enb`. Вместо `test-file-system` предполагается использовать пакет `mock-fs`. ([#365], [#369], [#371])
+* Удалена утилита `dir-glob`. В качестве замены предполагается пакет `glob`. ([#378])
+* Удалена утилита `drop-require-cache`, для совместимости с использующими её на момент релиза технологиями оставлена заглушка. Вместо `drop-require-cache` предполагается использовать модуль `clear-require`. ([#341], [#342])
+* Удалена утилита `async-require`, для совместимости с использующими её на момент релиза технологиями оставлена заглушка. Вместо `async-require` предполагается использовать модуль `enb-async-require`. ([#400])
+* Удалена утилита `require-or-eval`, для совместимости с использующими её на момент релиза технологиями оставлена заглушка. Вместо `require-or-eval` предполагается использовать модуль `enb-require-or-eval`. ([#400])
+* Удалены устаревшие методы из `FileList`. ([#363])
+* Обновлен `ENB server`: удалены избыточные зависимости, остальные обновлены до актуальных на момент релиза версий. ([#393])
+
```
Comment (j0tunn) [inline CHANGELOG.md]: забыл про https://github.com/enb-make/enb/pull/357

Comment (SwinX) [inline CHANGELOG.md]: Добавил

**Comment by j0tunn** [comment]
По мне :ok:

**Pull request merged**